flag_engine/identities/models.py

```diff
@@ -39,3 +39,8 @@ def update_traits(self, traits: typing.List[TraitModel]) -> typing.List[TraitMod
 
         self.identity_traits = list(existing_traits.values())
         return self.identity_traits
+
+    def prune_features(self, valid_features: typing.List[str]) -> None:
```

Comment:
Maybe this argument should be named `valid_feature_names` to be explicit?

Reply:
Yep, makes sense